pkg/printers/internalversion/printers.go

```diff
@@ -0,0 +1,69 @@
+package internalversion
+
+import (
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/duration"
+
+	clusterapis "github.com/karmada-io/karmada/pkg/apis/cluster"
+	"github.com/karmada-io/karmada/pkg/printers"
+)
+
+// AddHandlers adds print handlers for default Karmada types dealing with internal versions.
+func AddHandlers(h printers.PrintHandler) {
+	clusterColumnDefinitions := []metav1.TableColumnDefinition{
+		{Name: "Name", Type: "string", Format: "name", Description: metav1.ObjectMeta{}.SwaggerDoc()["name"]},
+		{Name: "Version", Type: "string", Description: "KubernetesVersion represents version of the member cluster."},
+		{Name: "Mode", Type: "string", Description: "SyncMode describes how a cluster sync resources from karmada control plane."},
+		{Name: "Ready", Type: "string", Description: "The aggregate readiness state of this cluster for accepting workloads."},
+		{Name: "Age", Type: "string", Description: metav1.ObjectMeta{}.SwaggerDoc()["creationTimestamp"]},
+	}
+	h.TableHandler(clusterColumnDefinitions, printClusterList)
```

Comment:
Here we may need to ignore errors because we enable `errcheck` golangci-lint.

Reply:
@XiShanYongYe-Chang fixed